### ISSUE TITLE
Not stopping playback in search results

### ISSUE THREAD

**Issue by duckbrain** [andrewbrey/netflix-tweaked#20]
**Describe the bug**
When viewing the main browse page, it blocks the video playback when hovering over a movie, but in search results it does not.

**To Reproduce**
1. Go to Netflix.com
2. Search for "test"
3. Hover over many of the video titles eg: Johny Test
4. See the videos playing

**Expected behavior**
I expect the thumbnails to expand and not play videos like they do on the main screen.

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: Chrome
 - Version: 74.0.3729.169

**Comment by andrewbrey** [comment]
Thanks for the issue report @duckbrain - the way the extension works today, only the home screen of Netflix is actually modified, so this is a feature enhancement to encompass more routes there.

I am about to be out of the country for a while, so I can't take a look at this enhancement just yet, but I'll get to it once I'm back.

Cheers!

**Comment by duckbrain** [comment]
Yeah, it's no rush. I just started using your extension. I'm assuming that means you would accept a PR to add that?

I just tried seeing if I could try enabling it for other pages to see if it would just work, but it seems [like Gulp 3 doesn't work on Node 12](https://github.com/gulpjs/gulp/issues/2324). I tried using Node 11 in a Docker container, but I'm still having build trouble. Would you like me to open another issue about that?

**Comment by andrewbrey** [comment]
I would absolutely welcome a PR! I don't _love_ some of the code, but I don't think it's too hard to follow on the whole.

As for the build - sure, if you could open up an issue for me to build a Dockerfile that has a working dev environment inside, that would be great! For reference, I believe I was on Node 10 the last time that I worked on this code (I believe 11 is not supported and 12 is not yet LTS, so 10 is probably a safe guess). If you don't mind doing more experimentation, I would love if you could test in a Node 10 container as well before abandoning ship, but no worries if you would prefer to just wait till I have time to get a Dockerfile added to the project.

I've also been meaning to modernize the build system anyway, so another option would be to figure out how to use Gulp 4 (I do not believe there's a migration path between 3 <-> 4, so that's likely a non-trivial amount of work) or alternatively use webpack or another tool to accomplish the same goals currently being serviced by gulp tasks (this is likely easier since Gulp isn't doing _all that much_ in this project).

If you decide to take a look at any of this, thanks! No worries whatsoever if not :)

**Comment by andrewbrey** [comment]
@duckbrain I just quickly threw together a `docker-compose.yml` that you can use with `docker-compose run --rm netflix-tweaked` - this will stand up a Node 10 environment that can run the gulp tasks. 

It uses a named volume to store the contents of the `node_modules` directory so that there's no conflicts with your host filesystem node_modules directory. Big caveat that I haven't thoroughly tested this as I had limited time to put it together!

Hopefully this is helpful for you in taking a look at this PR.

Cheers!
